common/customproperties.go

```diff
@@ -36,15 +33,7 @@ func (c CustomProperties) PropertiesList() []*custom_properties.CT_Property {
 	return c.x.Property
 }
 
-func (c CustomProperties) GetPropertyByName(name string) CustomProperty {
-	property := c.getPropertyByName(name)
-	if property == nil {
-		return nil
-	}
-	return CustomProperty(property)
-}
-
-func (c CustomProperties) getPropertyByName(name string) *custom_properties.CT_Property {
+func (c CustomProperties) GetPropertyByName(name string) *custom_properties.CT_Property {
```

Comment:
Cannot change signature of exported functions

Reply:
this was unexported and became exported, so we didn't change, only add one

Reply:
ok since it was not exported that is fine.